src/Conversion/ONNXToKrnl/ConvertONNXToKrnl.cpp

```diff
@@ -88,6 +97,16 @@ void FrontendToKrnlLoweringPass::runOnOperation() {
   // TODO: add any other ops which are considered legal.
   // Some operations can be marked as being still legal.
   // Example: target.addLegalOp<mlir::OpName>();
+  if (testRNNOps) {
```

Comment:
Can you add a comment on the approach. Let me say what I think it is. You enable ONNX operations to be generated, as you transform ONNX ops into a sequence of other ONNX ops that will then be later lowered and/or combined when operating on constants

Reply:
Done.